finetuner-workflow/finetune-workflow.yaml

```diff
@@ -109,31 +142,39 @@ spec:
     # Training GPU - A40, 48gb VRAM
     - name: trainer_gpu
       value: 'A40'
+    # Training number of GPUs
+    - name: trainer_gpus
+      value: 1
+    # Training number of CPU cores
+    - name: trainer_cores
+      value: 8
+    # Training compute RAM in gigabytes
+    - name: trainer_ram
+      value: 192
     # Inference GPU - RTX A5000, 24gb VRAM
     - name: inference_gpu
       value: 'RTX_A5000'
     # Container images -- generally, don't alter this.
     - name: model_downloader_image
       value: 'ghcr.io/wbrown/gpt_bpe/model_downloader'
     - name: model_downloader_tag
-      value: 'e2ef65f'
+      value: '73cceb0'
```

Comment:
Can we set these tags dynamically for all images build by us?

Reply:
No, we don't want to do that. We want to control when we update our dependencies.